src/Monolog/Logger.php

```diff
@@ -315,6 +321,7 @@ public function addRecord(int $level, string $message, array $context = []): boo
         }
 
         while ($handler = current($this->handlers)) {
+            $this->logMessageStack [] = $message;
```

Comment:
Thanks for your suggestion. I will write a test. Maybe the better idea would be to just get the last log without making it an array.